index.js

```diff
@@ -1,14 +1,6 @@
-const express = require('express');
-const helmet = require('helmet');
-
-const server = express();
-
-server.use(express.json());
-server.use(helmet());
-
-// endpoints here
+const server = require("./api/server.js");
 
 const port = 3300;
 server.listen(port, function() {
   console.log(`\n=== Web API Listening on http://localhost:${port} ===\n`);
```

Comment:
If you wanted, you could keep these lines in server.js and in your package.json scripts you could then change the "start" script to "nodemon server.js".